src/store/trees/trees.js

```diff
@@ -78,7 +78,7 @@ export default {
     isFeatureTreeDuplicateEnabled: ({ui}) => lodash.find(ui.enabledFeatures, (feature) => feature === 'treeDuplicate'),
     isFeatureTreeViewVersionsEnabled: ({ui}) => lodash.find(ui.enabledFeatures, (feature) => feature === 'treeViewVersions'),
     isFeatureTreeDuplicateOfEnabled: ({ui}) => lodash.find(ui.enabledFeatures, (feature) => feature === 'treeDuplicateOf'),
-    isResourceEditorEnabled: ({ui}) => lodash.find(ui.enabledFeatures, (feature) => feature === 'resourceEditor'),
+    isResourceViewerEnabled: ({ui}) => lodash.find(ui.enabledFeatures, (feature) => feature === 'resourceViewer'),
```

Comment:
renamed the feature to `resourceViewer` to reduce confusion, because we have component named `ResourceEditor.vue`